acceptance/setup/pre_suite/80_add_dev_repo.rb

```diff
@@ -1,4 +1,3 @@
-install_opts = options.merge( { :dev_builds_repos => ["PC1"] })
```

Comment:
This option was removed in https://github.com/puppetlabs/beaker/pull/1044/commits/e074def22605b7c5acfdb28a3877ac293547436e, which looks like it was released in beaker 2.32.0, so removing this should not break anything. I'd like to see the tests run before this is merged though